docs/conf.py

```diff
@@ -310,6 +311,12 @@
     'only_warn_on_example_error': True,
 }
 
+# sphinx tags
+tags_create_tags = True
+tags_overview_title = 'SunPy Examples Gallery Tags'
```

Comment:
```suggestion
tags_overview_title = 'sunpy examples gallery tags'
```
?

Reply:
This is the title of the page shouldn't it be title case?

Reply:
I guess but it looks ugly.